src/app/static/src/app/generated.d.ts

```diff
@@ -28,6 +28,50 @@ export type AncResponseData = {
   art_coverage?: number;
   [k: string]: any;
 }[];
+export interface BarchartIndicator {
```

Comment:
It seems the typings were actually quite out of date, so lots of unrelated changes here!